### DISCUSSION TITLE
Import recipes from Samsung Food

### DISCUSSION THREAD

**Discussion by cristobal-io** [mealie-recipes/mealie#4349] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Importing from https://samsungfood.com/ doesn't work

### Please provide a concise description of the feature that would resolve your issue.

Allow importing from https://samsungfood.com/ scrapping the site, per example it has the items `data-testid="recipe-ingredient"` and the steps have also its own section `data-scroll="instructions"`

### Please consider and list out some caveats or tradeoffs made in your design decision

This would be beneficial as the popularity of the service increases and it allows for easy import on instagram recipes.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
Just to make sure, because it seems samsung foods may support both cases. 
Do you mean importing like in scraping a single recipe at a time, or do you mean importing a bunch or recipes in a collection exported by samsung foods into mealie as a whole.

**Reply by cristobal-io** [reply]
Hi,

It would be best if you could bulk import a collection, but I have not seen an option on Samsung foods to export in batch, so I was talking individually, but personally any option works. 

Thanks for the follow up
